src/index.html

```diff
@@ -0,0 +1,271 @@
+    <div class="progress-bar__bar" style="width: 14%"></div>
+  </div>
+  
+  <section id="main" class="main">
+    <div class="main__wrapper">
+      <div class="main__content">
+        <div class="title">How many bedrooms in the property?</div>
+        <div class="input-container ">
+          <input type="text" class="input input--range" placeholder="2 bedrooms">
+          <div class="input-button input-button--disabled">-</div>
+          <div class="input-button">+</div>
+        </div>
+        <div class="title">How many bathrooms in the property?</div>
+        <div class="input-container">
+          <input type="text" class="input input--range" placeholder="1 bedroom">
+          <div class="input-button input-button--disabled">-</div>
+          <div class="input-button">+</div>
+        </div>
+        
+        <div class="title">Do you have a living room?</div>
+        <div class="button button--secondary">Add living room</div>
+        
+        <div class="title">Which amenities does your property have?</div>
+        <div class="input-container input-container--bordered input-container--focused">
+          <div class="checkbox checkbox--checked"></div>
+          <label for="">WiFi</label>
+        </div>
+        <div class="input-container input-container--bordered">
+          <div class="checkbox"></div>
+          <label for="">Cable TV</label>
+        </div>
+        <div class="input-container input-container--bordered">
+          <div class="checkbox"></div>
+          <label for="">Heating</label>
+        </div>
+        <div class="input-container input-container--bordered">
+          <div class="checkbox"></div>
+          <label for="">Air conditioning</label>
+        </div>
+        <div class="input-container input-container--bordered">
+          <div class="checkbox"></div>
+          <label for="">Outdoor area (terrace, balcony)</label>
+        </div>
+        <div class="input-container input-container--bordered">
+          <div class="checkbox"></div>
+          <label for="">Elevator</label>
+        </div>
+
+        <div class="title">Does the property have gas installation?</div>
+        <div class="input-container input-container--bordered input-container--focused">
+          <div class="radio-button radio-button--checked"></div>
+          <label for="">Yes, piped gas</label>
+        </div>
+        <div class="input-container input-container--bordered">
+          <div class="radio-button"></div>
+          <label for="">Yes, bottled gas</label>
+        </div>
+        <div class="input-container input-container--bordered">
+          <div class="radio-button"></div>
+          <label for="">No, it's all electric</label>
+        </div>
+        
+        <div class="title">Describe your property</div>
+        <textarea name="" id="" rows="5" class="textarea" placeholder="I.E: Two bedroom apartment with a balcony and great view in trendy Bairro Alto."></textarea>
```

Comment:
Please disregard structure implemented in the demo as it acts only as a placeholder.